web_toggle/static/src/js/bs_toggle.js

```diff
@@ -0,0 +1,74 @@
+openerp.web_toggle = function(instance, local) {
+    var _t = instance.web._t,
+        _lt = instance.web._lt;
```

Comment:
I haven't seen where you are using those variables. If you don't need it, you can remove it;)